{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -1,3 +1,54 @@
+[build-system]
+requires = ["setuptools>=62.0", "setuptools-git-versioning<2"]
```

Comment:
is this also supposed to be built dynamically? so we get latest setuptools?  Or we want to freeze it?

Reply:
We could do if statements (see [https://github.com/scientific-python/cookie/blob/main/%7B%7Bcookiecutter.project_name%7D%7D/pyproject.toml](url)), but I don't think it's built dynamically.

Reply:
In theory, we will get the latest `setuptools`. However, it may be possible that future builds of `python-build` might use caching (e.g. if the user has setuptools `61.0` it will use that). We definitely want to guarantee higher than `62.0`.

As for `setuptools-git-versioning`, version `2.0.0` came out two months ago, if we want to test that for our builds. Otherwise, let's keep it frozen below version 2.

Reply:
> In theory, we will get the latest `setuptools`. However, it may be possible that future builds of `python-build` might use caching (e.g. if the user has setuptools `61.0` it will use that). We definitely want to guarantee higher than `62.0`.
> 
Let's leave that as static then.  Good as is.